### ISSUE TITLE
MySQL change and drop column support (unparsable section error)

### ISSUE THREAD

**Issue by MontealegreLuis** [sqlfluff/sqlfluff#1628]
## Expected Behaviour

SQLFluff reporting issues on change and drop column statements

## Observed Behaviour

SQLFluff will report unparsable sections on drop and change column statements

### Drop column

- Create file `drop.sql`

```sql
ALTER TABLE a_table
DROP COLUMN a_column;
```
- Run SQLFluff lint command 

```bash
sqlfluff lint --dialect=mysql drop.sql
```

- Find error message below

```
== [drop.sql] FAIL
L:   2 | P:   6 | L014 | Unquoted identifiers must be consistently lower case.
L:   2 | P:  12 |  PRS | Line 2, Position 12: Found unparsable section: '
                       | a_column'
All Finished 📜 🎉!
```

### Change column

- Create file `change.sql`

```sql
ALTER TABLE a_table
CHANGE COLUMN old_column new_column INT(11) NULL DEFAULT NULL;
```
- Run SQLFluff lint command 

```bash
sqlfluff lint --dialect=mysql change.sql
```

- Find error message below

```
== [change.sql] FAIL
L:   2 | P:   8 | L014 | Unquoted identifiers must be consistently lower case.
L:   2 | P:  14 |  PRS | Line 2, Position 14: Found unparsable section: '
                       | old_column new_column INT(11) NULL DEFA...'
All Finished 📜 🎉!
```

## Dialect

MySQL

## Version

```
Python 3.9.7
pip 21.2.4 from /usr/local/lib/python3.9/site-packages/pip (python 3.9)
sqlfluff, version 0.6.9
```

## Configuration

```ini
[sqlfluff:rules:L004]
indent_unit = space

[sqlfluff:rules:L010]
capitalisation_policy = upper

[sqlfluff:rules:L030]
capitalisation_policy = upper

[sqlfluff:rules:L040]
capitalisation_policy = upper
```

**Comment by WittierDinosaur** [comment]
Docs: https://dev.mysql.com/doc/refman/8.0/en/alter-table.html
